identity-wallet/src/state/reducers/history.rs

```diff
@@ -0,0 +1,31 @@
+use serde::{Deserialize, Serialize};
+use ts_rs::TS;
+
+#[derive(Clone, Serialize, Deserialize, Debug, PartialEq, Eq, TS)]
+#[ts(export)]
+pub enum EventType {
+    #[serde(rename = "added_connection")]
+    AddedConnection,
```

Comment:
The [naming convention for events](https://www.eventstore.com/blog/whats-in-an-event-name#naming-conventions) is **SubjectObjectPredicate** so: `AddedConnection` --> `ConnectionAdded`

Reply:
Ah ok I will rename :+1: